packages/block-editor/src/components/grid-visualizer/utils.js

```diff
@@ -0,0 +1,5 @@
+export function getComputedCSS( element, property ) {
```

Comment:
This is the third instance of this function in the block-editor package, might be worth consolidating (as a separate task)

Reply:
The refactor in https://github.com/WordPress/gutenberg/pull/59227 brings it down to 2! Getting close... 😂